redbot/cogs/downloader/downloader.py

```diff
@@ -494,12 +494,23 @@ async def pipinstall(self, ctx: commands.Context, *deps: str) -> None:
 
         - `<deps...>` The package or packages you wish to install.
         """
+        confirmation_msg = await ctx.send(f"Are you sure you want to install `{(', '.join(deps))}`?")
```

Comment:
You are missing translation support for this string.

Red also has a `humanize_list` utility function which should be used when showing a list to end-users